Dockerfile

```diff
@@ -1,54 +1,65 @@
+    && unzip /tmp/consul -d /usr/local/bin \
+    && rm /tmp/consul.zip
+
+# Consul client
+RUN export CONSUL_CLIENT_CHECKSUM=037150d3d689a0babf4ba64c898b4497546e2fffeb16354e25cef19867e763f1 \
+    && curl -Lso /tmp/consul-cli.tgz "https://github.com/CiscoCloud/consul-cli/releases/download/v${CONSUL_CLI_VER}/consul-cli_${CONSUL_CLI_VER}_linux_amd64.tar.gz" \
+    && echo "${CONSUL_CLIENT_CHECKSUM}  /tmp/consul-cli.tgz" | sha256sum -c \
+    && tar zxf /tmp/consul-cli.tgz -C /usr/local/bin --strip-components 1 \
+    && rm /tmp/consul-cli.tgz
 
-RUN export CONTAINERPILOT_CHECKSUM=e7973bf036690b520b450c3a3e121fc7cd26f1a2 \
+# Add ContainerPilot and set its configuration file path
+RUN export CONTAINERPILOT_CHECKSUM=e886899467ced6d7c76027d58c7f7554c2fb2bcc \
     && curl -Lso /tmp/containerpilot.tar.gz \
-         "https://github.com/joyent/containerpilot/releases/download/${CONTAINERPILOT_VER}/containerpilot-${CONTAINERPILOT_VER}.tar.gz" \
+        "https://github.com/joyent/containerpilot/releases/download/${CONTAINERPILOT_VER}/containerpilot-${CONTAINERPILOT_VER}.tar.gz" \
     && echo "${CONTAINERPILOT_CHECKSUM}  /tmp/containerpilot.tar.gz" | sha1sum -c \
     && tar zxf /tmp/containerpilot.tar.gz -C /usr/local/bin \
     && rm /tmp/containerpilot.tar.gz
 
+# Add our configuration files and scripts
+COPY /etc/containerpilot.json /etc/containerpilot.json
+COPY /etc/elasticsearch.yml /usr/share/elasticsearch/config/elasticsearch.yml
+COPY /bin/* /usr/local/bin/
+
+# Should we remove unzip?
+# RUN apk del unzip tar
+
 # Create and take ownership over required directories
-RUN mkdir -p /var/lib/elasticsearch/data && \
-    chown -R elasticsearch:elasticsearch /var/lib/elasticsearch/data && \
-    chown -R root:elasticsearch /etc/elasticsearch && \
-    chmod g+w /etc/elasticsearch
+RUN mkdir -p /opt/consul/config \
+    && mkdir -p /opt/consul/data \
+    && chmod 770 /opt/consul/data \
+    && chown -R elasticsearch:elasticsearch /opt/consul \
+    && mkdir -p /etc/containerpilot \
+    && chmod -R g+w /etc/containerpilot \
+    && chmod +x /usr/local/bin/elastic-server.sh \
+    && chown -R elasticsearch:elasticsearch /etc/containerpilot
 
+# back to elastic USER
 USER elasticsearch
 
-# Add our configuration files and scripts
-COPY /etc/containerpilot.json /etc
-COPY /etc/elasticsearch.yml /etc/elasticsearch/elasticsearch.yml
-COPY /bin/manage.sh /usr/local/bin
-
 # Expose the data directory as a volume in case we want to mount these
 # as a --volumes-from target; it's important that this VOLUME comes
 # after the creation of the directory so that we preserve ownership.
-VOLUME /var/lib/elasticsearch/data
-
-# We don't need to expose these ports in order for other containers on Triton
-# to reach this container in the default networking environment, but if we
-# leave this here then we get the ports as well-known environment variables
-# for purposes of linking.
-EXPOSE 9200
-EXPOSE 9300
```

Comment:
Yes, the Dockerfile includes them.
https://github.com/elastic/elasticsearch-docker/blob/master/build/elasticsearch/Dockerfile#L44